blazor/common/data-binding/restful-service-binding.md

```diff
@@ -314,72 +322,107 @@ app.Run();
 
-![Manage Nuget Packages in Blazor](../images/odata-manage-nuget.png)
+If you utilize `WebAssembly or Auto` render modes in the Blazor Web App need to be install Syncfusion Blazor components NuGet packages within the client project.
 
-Now, in the **Browse** tab, search and install the **Syncfusion.Blazor.Grid** NuGet package.
+Alternatively, you can utilize the following package manager command to achieve the same.
 
-![Add Syncfusion package in Blazor](../images/odata-syncfusion-package.png)
+{% tabs %}
+{% highlight C# tabtitle="Package Manager" %}
+
+Install-Package Syncfusion.Blazor.Grid -Version {{ site.releaseversion }}
+Install-Package Syncfusion.Blazor.Themes -Version {{ site.releaseversion }}
 
-N> For this demo, Syncfusion.Blazor(**19.1.0.66**) NuGet package is used. A new **Syncfusion.Blazor** NuGet package with new enhancement has been released in our every-week release and main release. So, you can check and update to the latest versions by using this [link](https://www.nuget.org/packages/Syncfusion.Blazor).
+{% endhighlight %}
+{% endtabs %}
 
-Open **_Import.razor** file and add the following namespaces which are required to use Syncfusion Blazor components in this application.
+N> Syncfusion Blazor components are available in [nuget.org](https://www.nuget.org/packages?q=syncfusion.blazor). Refer to [NuGet packages](https://blazor.syncfusion.com/documentation/nuget-packages) topic for available NuGet packages list with component details.
 
-{% tabs %}
-{% highlight razor tabtitle="_Import.razor" %}
+Open **~/_Imports.razor** file and import the following namespace.
+
+{% highlight razor %}
 
 @using Syncfusion.Blazor
 @using Syncfusion.Blazor.Grids
-@using Syncfusion.Blazor.Data
-@using ODataServiceProject.Models
 
 {% endhighlight %}
-{% endtabs %}
 
-Refer to [Register Services](https://blazor.syncfusion.com/documentation/getting-started/blazor-server-side-visual-studio#register-syncfusion-blazor-service) and add the Syncfusion service in the application.
+Now, register the Syncfusion Blazor Service in the **~/Program.cs** file of your App.
+
+For a Blazor Web App with `WebAssembly` or `Auto (Server and WebAssembly)` interactive render mode, register the Syncfusion Blazor service in both **~/Program.cs** files of your web app.
+
+```cshtml
+
+....
+using Syncfusion.Blazor;
+....
+builder.Services.AddSyncfusionBlazor();
+....
+
+```
 
 Themes provide life to components. Syncfusion Blazor has different themes. They are:
 
-* Bootstrap4
-* Material
-* Office 365
+* Fabric
 * Bootstrap
+* Material
 * High Contrast
 
-In this demo application, the **Bootstrap4** theme will be used.
+In this demo application, the Bootstrap theme will be used.
 
-* For **.NET 6** app, add theme in the `<head>` of the **~/Pages/_Layout.cshtml** file.
+  * For **Blazor Web App**,  refer style sheet inside the `<head>` of **~/Components/App.razor** file for .NET 8.
```

Comment:
Style sheet is single word

Reply:
`stylesheet ` word updated in existing document.